### ISSUE TITLE
Data

### ISSUE THREAD

**Issue by ucasyjz** [ZhenbangDu/Reliable_AD#4]
Where is the training data with image, i only find the text in txt file. Another question, will the RFNet weight will be released? and the effectiveness of the depth and salience in RFNet? Thank you very much

**Comment by ZhenbangDu** [comment]
Hi! Thank you very much for your interest in our work. The training data images are provided in compressed packages, with labels for each sample included in the text file.

At this time, we don’t plan to release the RFNet weights, but you can find the training details and model structure in the paper. Additionally, the effectiveness of the depth and salience components in RFNet is presented in Table 2 of our paper.

Thank you again for your attention!
